code/src/App.js

```diff
@@ -22,9 +22,9 @@ const renderAlbum = album => {
 
 export const App = () => {
   return (
-    <div>
+    <Fragment>

```

Comment:
You can also use `<>` and `</>` instead of `<Fragment>`

Reply:
It won't be rendered but it's necessary for React if you have two or more components.